backend/build.gradle.kts

```diff
@@ -252,6 +254,56 @@ tasks {
             into("./LegalDocML.de")
         }
     }
+
+    register("processDBFixtures") {
+        var baseDir = fileTree("src/main/resources/db/data")
+        var norms =
+            baseDir.matching {
+                include("**/rechtsetzungsdokument-1.xml")
+            }
+
+        inputs.files(norms)
+
+        var outdir = layout.buildDirectory.dir("resources/main/db/data")
+        outputs.dir(outdir)
+
+        doLast {
+            norms.files.forEach { rechtsetzungsdokumentFile ->
+                var relativeNormFolder = rechtsetzungsdokumentFile.parentFile.relativeTo(baseDir.dir)
+                logger.info("Creating db migration for $relativeNormFolder")
+                var dbMigrationFile = file(outdir.get().file("$relativeNormFolder.sql"))
+
+                if (dbMigrationFile.exists()) {
+                    dbMigrationFile.delete()
+                }
+                dbMigrationFile.parentFile.mkdirs()
+                dbMigrationFile.createNewFile()
+
+                var rechtsetzungsdokumentContent = rechtsetzungsdokumentFile.readText(Charsets.UTF_8)
+                // we just look for the first thing looking like a norm expression eli as it should always be the one of the norm, and we don't need to parse the xml this way
+                var eliNormExpression =
+                    "eli/bund/[-a-z0-9]+/\\d{4}/(s[0-9]+[a-zäöüß]*|[0-9]+(-\\d+)?)/\\d{4}-\\d{2}-\\d{2}/\\d+/[a-z]{3}"
+                        .toRegex()
+                        .find(
+                            rechtsetzungsdokumentContent,
+                        )?.value
```

Comment:
Do we want to throw an error and not have them silently like when no eli is found because we had a completely wrong XML?